docs/examples/simple_interpolation.rst

```diff
@@ -49,12 +49,21 @@ which can be "loaded" into Sherpa using the
    >>> d = Data1D('interpolation', x, y)
    >>> print(d)
    name      = interpolation
-   x         = [1, 1.5, 2, 4, 8, 17]
-   y         = [1, 1.5, 1.75, 3.25, 6, 16]
+   x         = Float64[6]
+   y         = Float64[6]
    staterror = None
    syserror  = None
    None
-   
+
+.. note::
+
+   Creating the :py:class:`~sherpa.data.Data1D` object will - from
+   Sherpa 4.12.0 - warn you that it is converting the dependent axis
+   data to NumPy. That is, the above call will cause the following
+   message to be displayed::
+
+        UserWarning: Converting array [1, 1.5, 1.75, 3.25, 6, 16] to numpy array
```

Comment:
Unless I miss something here it seems that the changes in line 52 and 53 will change this to be just `[6]`. I don't understand the changes in 52 and 53 and the fact that it's not consistent with this new doc note indicates that maybe this line is correct and the 52/53 are not?

Reply:
I don't understand what you mean - but I am too close to this to read it properly. I am just trying to note that the warning we see in the example below is new:

```
>>> from sherpa.data import Data1D
>>> d = Data1D('test', [1, 1.5, 2, 4, 8, 17], [1, 1.5, 1.75, 3.25, 6, 16])
/lagado.real/sherpa/sherpa-master/sherpa/data.py:49: UserWarning: Converting array [1, 1.5, 1.75, 3.25, 6, 16] to numpy array
  warnings.warn("Converting array {} to numpy array".format(array))
>>> print(d)
name      = test
x         = Float64[6]
y         = Float64[6]
staterror = None
syserror  = None
>>> 
```

Reply:
Ah, now I get it. I thought `Float64[6]` means a single float with value 6, but it is in fact six floats in a numpy array. Confusing, but that's not the point of this PR, so consider this solved for now.